SingularityService/src/main/java/com/hubspot/singularity/views/IndexView.java

```diff
@@ -68,6 +74,15 @@ public IndexView(String singularityUriBase, String appRoot, SingularityConfigura
     this.finishedTaskLogPath = configuration.getUiConfiguration().getFinishedTaskLogPath();
 
     this.commonHostnameSuffixToOmit = configuration.getCommonHostnameSuffixToOmit().or("");
+
+    ObjectWriter ow = new ObjectMapper().writer().withDefaultPrettyPrinter();
```

Comment:
instead of instantiating a new `ObjectMapper` each time here, inject one in `UiResource` and pass it through the `IndexView` constructor